src/test/resources/features/LevelPlacement.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Level Placement after evaluation test
+  After the student has completed an evaluation test and the score is calculated the student is placed into the level
+
+  Scenario: Student with score "Score" is placed into the level "Level"
```

Comment:
Scenarios with multiple values have a slightly different syntax, here you have an example: https://www.baeldung.com/cucumber-scenario-outline